components/sendgrid/actions/add-email-to-global-supression/add-email-to-global-supression.js

```diff
@@ -0,0 +1,23 @@
+const common = require("../common");
+
+module.exports = {
+  ...common,
+  key: "sendgrid-add-email-to-global-supression",
+  name: "Add Email To Global Supression",
+  description:
+    "Allows you to add one or more email addresses to the global suppressions group. [See the docs here](https://sendgrid.api-docs.io/v3.0/suppressions-global-suppressions/add-recipient-addresses-to-the-global-suppression-group)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ...common.props,
+    recipientEmails: {
+      type: "string[]",
+      label: "Recipient Emails",
+      description:
+        "An string array of email addresses to be added to the global suppressions group. Example `[\"email1@example.com\",\"email2@example.com\"]`",
+    },
+  },
+  async run() {
+    return this.sendgrid.addEmailToGlobalSupression(this.recipientEmails);
+  },
```

Comment:
Need to include the `$` variable and export a summary for all actions.
```suggestion
  async run({ $ }) {
    const resp = await this.sendgrid.addEmailToGlobalSupression(this.recipientEmails);
    $.export("$summary", "Successfully added emails to global supressions group");
    return resp;
  },
```